CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ version directory, and  then changes are introduced.
 
 ## [Unreleased]
 
+## [v6.1.0] 2020-05-06
```

Comment:
Decided to go with `6.1.0` because this is the first flattened release of k8scc.